fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/ambassador/AbstractSumoAmbassador.java

```diff
@@ -1368,7 +1368,11 @@ private void initializeTrafficLights(long time) throws InternalFederateException
         Map<String, Collection<String>> trafficLightGroupLaneMap = new HashMap<>();
         for (String trafficLightGroupId : trafficLightGroupIds) {
             try {
-                trafficLightGroups.add(bridge.getTrafficLightControl().getTrafficLightGroup(trafficLightGroupId));
+                TrafficLightGroup trafficLightGroup = bridge.getTrafficLightControl().getTrafficLightGroup(trafficLightGroupId);
+                if (trafficLightGroup == null) {  // FIXME: workaround for railway signals
```

Comment:
remove "FIXME" from comment. Better write "reading traffic light groups does not work for railway signals"